wof/WaterML_1_1.py

```diff
@@ -402,7 +402,7 @@ def quote_xml(inStr):
         pos = mo.end()
     s3 = s1[pos:]
     s2 += quote_xml_aux(s3)
-    return s2
+    return s2.decode(ExternalEncoding)
```

Comment:
Everything that uses `quote_xml` encodes the string. I get decode errors if I didn't decode them. 😞